rewrite-test/src/main/kotlin/org/openrewrite/java/JavaTemplateTest.kt

```diff
@@ -812,6 +812,73 @@ interface JavaTemplateTest : JavaRecipeTest {
         """
     )
 
+    @Test
+    fun addVariableAnnotationsToVariableAlreadyAnnotated(jp: JavaParser) = assertChanged(
+        jp,
+        recipe = object : JavaIsoVisitor<ExecutionContext>() {
+            val t = JavaTemplate.builder({ cursor }, "@Deprecated")
+                .doBeforeParseTemplate(print)
+                .build()
+
+            override fun visitVariableDeclarations(multiVariable: J.VariableDeclarations, p: ExecutionContext): J.VariableDeclarations {
+                if(multiVariable.leadingAnnotations.size == 1) {
+                    return multiVariable.withTemplate(t, multiVariable.coordinates.addAnnotation(comparing { it.simpleName }))
+                }
+                return super.visitVariableDeclarations(multiVariable, p)
+            }
+        }.toRecipe(),
+        before = """
+            class Test {
+                void test() {
+                    @SuppressWarnings("ALL") int m;
+                    final int n;
+                }
+            }
+        """,
+        after = """
+            class Test {
+                void test() {
+                    @SuppressWarnings("ALL")
+                    @Deprecated int m;
```

Comment:
I think here the formatting could be improved and add a linebreak after `@Deprecated`...
For our current requirement, the result is 'good enough' though. 
I can give it a try tomorrow if yo want me to?

Reply:
Yes, that would be nice. Ultimately, that blank line is the `suffix` on the last `Comment`

Reply:
I might not have understood, but I couldn't see how "blank line is the suffix on the last Comment" should work?

````
/** 
* some comment
*/
int a;
````
became 
````
/** 
* some comment
*/
@Deprecated int a;
````
and without an existing comment, I didn't get it?

However, I got most cases working and tested. I would have found a dedicated test class for formatting related Visitors  helpfull, but that's for another time ;)

Reply:
I would call it "done" (for now)? ;)